resources/bundles/org.eclipse.core.filesystem/src/org/eclipse/core/internal/filesystem/messages.properties

```
@@ -24,7 +24,8 @@ deleteProblem = Problems encountered while deleting files.
 deleting = Deleting: {0}.
 failedCreateAccessDenied=Cannot create file, access denied: {0}.
 failedCreateWrongType=Cannot create file because existing file of wrong type exists: {0}.
-failedMove = Critical failure moving: {0} to: {1}. Content is lost.
+failedMove = Critical failure moving '{0}' to '{1}'. Content is lost.
+failedCopy = Could not copy file '{0}' to '{1}'
```

To align the style to the other messages in this file:

```suggestion
failedCopy = Could not copy file '{0}' to '{1}'.
```

---

Thanks. Done.